test/module/vendor/rocksdb_test.cpp

```diff
@@ -0,0 +1,273 @@
+  {
+    RocksDbCommon common(tx_context_);
+    common.encode(55ull);
+    ASSERT_TRUE(common.put("{}", "123").ok());
+  }
+  uint64_t value;
+  {
+    RocksDbCommon common(tx_context_);
+    ASSERT_TRUE(common.get("{}", "123").ok());
+    common.decode(value);
+  }
+  ASSERT_TRUE(value == 55ull);
+}
+
+TEST_F(RocksDBTest, Quorum) {
+  RocksDbCommon common(tx_context_);
+  forQuorum<kDbOperation::kCheck, kDbEntry::kMustNotExist>(
+      common,
+      "acc",
+      "dom",
+      [](auto /*account*/, auto /*domain*/, auto /*opt_quorum*/) {});
+
+  common.encode(5);
+  forQuorum<kDbOperation::kPut>(
+      common,
+      "acc",
+      "dom",
+      [](auto /*account*/, auto /*domain*/, auto /*opt_quorum*/) {});
+
+  forQuorum<kDbOperation::kGet, kDbEntry::kMustExist>(
+      common, "acc", "dom", [](auto account, auto domain, auto opt_quorum) {
+        ASSERT_EQ(account, "acc");
+        ASSERT_EQ(domain, "dom");
+        ASSERT_TRUE(opt_quorum);
+        ASSERT_EQ(*opt_quorum, 5);
+      });
+}
+
+TEST_F(RocksDBTest, Signatories) {
+  RocksDbCommon common(tx_context_);
+  auto cmd_check = [&](std::string_view pk) {
+    forSignatory<kDbOperation::kCheck, kDbEntry::kMustNotExist>(
+        common,
+        "acc",
+        "dom",
+        pk,
+        [](auto /*account*/, auto /*domain*/, auto /*opt_quorum*/) {});
+  };
+
+  auto cmd_put = [&](std::string_view pk) {
+    common.valueBuffer() = pk;
+    common.valueBuffer() += std::string_view{"_test"};
+    forSignatory<kDbOperation::kPut>(
+        common,
+        "acc",
+        "dom",
+        pk,
+        [](auto /*account*/, auto /*domain*/, auto /*opt_quorum*/) {});
+  };
+
+  cmd_check("pubkey1");
+  cmd_check("pubkey2");
+  cmd_check("pubkey3");
+
+  cmd_put("pubkey1");
+  cmd_put("pubkey2");
+  cmd_put("pubkey3");
+
+  int counter = 0;
+  enumerateKeysAndValues(common,
+                         [&](auto key, auto value) {
+                           if (key.ToStringView() != "pubkey1"
+                               && key.ToStringView() != "pubkey2"
+                               && key.ToStringView() != "pubkey3")
+                             throw;
+
+                           if (key.ToStringView() == "pubkey1"
+                               && value.ToStringView() != "pubkey1_test")
+                             throw;
+                           if (key.ToStringView() == "pubkey2"
+                               && value.ToStringView() != "pubkey2_test")
+                             throw;
+                           if (key.ToStringView() == "pubkey2"
+                               && value.ToStringView() != "pubkey2_test")
+                             throw;
+
+                           ++counter;
+                           return true;
+                         },
+                         fmtstrings::kPathSignatories,
+                         "dom",
+                         "acc");
+
+  ASSERT_EQ(counter, 3);
+}
+
+TEST(RocksDBTestCommon, Usage) {
```

Comment:
While this test case is a vendor test, because it does not use anything from iroha directory.